app/views/layouts/_flash_msg.html.haml

```diff
@@ -24,6 +24,10 @@
               %span.pficon.pficon-warning-triangle
               %span.pficon.pficon-warning-exclamation
             %strong= h(fl[:message])
+        - when :info
+          .alert.alert-info
```

Comment:
@romanblanco Use the info_msg partial here: manageiq/app/views/layouts/_info_msg.html.haml